Makefile

```diff
@@ -85,158 +95,232 @@ help: ## Display this help.
 ##@ Development
 
 .PHONY: manifests
-manifests: controller-gen kustomize yq ## Generate manifests e.g. CRD, RBAC etc.
+manifests: controller-gen yq kustomize ## Generate WebhookConfiguration, ClusterRole and CustomResourceDefinition objects.
 	$(CONTROLLER_GEN) rbac:roleName=manager-role crd:maxDescLen=0 webhook paths="./..." output:crd:artifacts:config=config/crd/bases
-	$(KUSTOMIZE) build config/helm-crd | yq --split-exp '"charts/astarte-operator/templates/crds/" + .metadata.name + ".yaml"' --no-doc
+	$(KUSTOMIZE) build config/helm-crd | $(YQ) --split-exp '"charts/astarte-operator/templates/crds/" + .metadata.name + ".yaml"' --no-doc
+	# ensure we're checking if crds shall be installed
+	@sed -i '1i{{- if .Values.installCRDs }}' charts/astarte-operator/templates/crds/* # prepend to each and every crd
+	@sed -i '$$a{{- end }}' charts/astarte-operator/templates/crds/* # append to the end of each and every crd
```

Comment:
If we encapsulate every CRD in this, what would happen when installing a CRD without Helm?
This seems an Helm-specific workflow, but `make manifests` in general might not be related to helm

Reply:
This is targeting only files contained in `charts/astarte-operator/templates`. Therefore, this is relevant only in the Helm scenario. The `make install` target ensures that we can install CRDs without go templates.